rpm/389-ds-base.spec.in

```diff
@@ -62,6 +62,7 @@ Group:            System Environment/Daemons
 BuildRoot:        %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 Obsoletes:        %{name} <= 1.4.0.9
 Obsoletes:        %{name}-legacy-tools
+Obsoletes:        %{name}-legacy-tools-debuginfo
```

Comment:
Just noticed that unversioned Obsoletes gives a warning:
```
warning: line 64: It's not recommended to have unversioned Obsoletes: Obsoletes:        389-ds-base-legacy-tools
warning: line 65: It's not recommended to have unversioned Obsoletes: Obsoletes:        389-ds-base-legacy-tools-debuginfo
```

Reply:
I did not see this warning, but I'll tweak it...